### ISSUE TITLE
RHCIDeployment entity has bad fields

### ISSUE THREAD

**Issue by Ichimonji10** [SatelliteQE/nailgun#112]
The `RHCIDeployment` entity defines its fields liks so:

```python
            'name': entity_fields.StringField(required=True),
            'organization_id': entity_fields.IntegerField(required=True),
            'lifecycle_environment_id': entity_fields.IntegerField(
                required=True),
            'deploy_rhev': entity_fields.BooleanField(required=True),
            'rhev_engine_host_id': entity_fields.IntegerField(required=True),
            'rhev_storage_type': entity_fields.StringField(required=True),
            'rhev_engine_admin_password': entity_fields.StringField(),
        }
```

There's two issues here:

* "_id" should not be appended to the end of field names. "_id", "_ids", "s" and other suffixes are all appended to field names in certain contexts (GETting, PUTting, POSTing). The base field names should not have *any* suffix.
* `IntegerField`s are used to describe foreign key relationships. That's totally wrong.

**Comment by Ichimonji10** [comment]
This bug will be more easily fixed if #110 is implemented first.

**Comment by JacobCallahan** [comment]
we could change the fields to deployment, organizations, lifecycle_environments, discovered hosts. All but deployment are one-to-many fields.
